third_party/src/default/assets/js/main.js

```diff
@@ -428,17 +428,17 @@ function renderSimpleHTMLRecursive(obj, package, spacing) {
                 shownPackages.push(package);
             }
             var href = '';
-            if (window.location.href.indexOf('modules') == -1) {
+            if (window.location.href.indexOf('/modules/') == -1) {
                 href = 'modules/';
             }
-            if (window.location.href.indexOf('assets') > -1 ||
-                window.location.href.indexOf('classes') > -1 ||
-                window.location.href.indexOf('enums') > -1 ||
-                window.location.href.indexOf('interfaces') > -1) {
+            if (window.location.href.indexOf('/assets/') > -1 ||
```

Comment:
@Fleker This code will be more accurate for path with classes. Imagining a path "127.0.0.1/modules/foo_classes/foo.html"

Reply:
That would be an odd case, but I understand that it may happen. Can you make this change to the `NavigationAccordion.ts` as that's the original file.